src/Exception/ObjectGeneratorPersisterException.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Fidry\AliceDataFixtures package.
+ *
+ * (c) Théo FIDRY <theo.fidry@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Fidry\AliceDataFixtures\Exception;
+
+class ObjectGeneratorPersisterException extends \Exception
+{
+    public static function entityMissingAssignedIdForField($entity)
+    {
+        return new self("Entity of type " . get_class($entity) . " is missing an assigned ID. " .
```

Comment:
You can use doc blocks here:

```
No ID found for the entity "EntityName". If this entity has an auto ID generator, this may be due to having it disabled because once instance of the entity had an an ID assigned. Either remove this assigned ID to allow the auto ID generator to operate or generate and ID for all the "EntityName" entities.
```